ARC-Seal: i=1; a=rsa-sha256; d=naver.com; s=arc-20180730; t=1750245309;
	cv=none; b=L9UUAYAwmRzwRQ8EhKnnH5KLSn4ugS4CXX3gcdQSPrSkRuOI+pAS7FlJ940E
	 xp+dM/gIN31LEphMDEshXa5mZGCim3NkP29UcTF8/taor4RcEHe6+YVb3yg7swuhemACX/
	 xfOsU/J6o10T5SRemrLiHX+qQ6CcMVkUtpXsNbLf7D/ElfHM1vtKAkZko61BXbbMh+CKf/
	 ZnwKrVdF3csgZ4n0NmxyEOzyvDAFwiw+OoG23Tt7gZG4ZFYorwV+DNGDevFZRfwvI9VJcL
	 jZ7m7tATJ7oL+MMGCtn+k/7Ahd/ZT5JEFIIEXmbFJ1AYDxC9D9dKBG+S/qNHMlmjlX0jeS
	 TQ==
ARC-Message-Signature: i=1; a=rsa-sha256; d=naver.com; s=arc-20180730;
	t=1750245309; c=relaxed/relaxed;
	bh=UU2pyKWcL22myledRTIjWFL5EhBsM4P4oVJOu/Hxj2A=;
	h=dkim-signature:from:to:subject:date:message-id; b=NOMqAQ8rErbrydkJY5s
	 +umVEi/k8wAJXPStvfkobBjwHuQvzga8ZEWSlthe5NZiKf7VvZnzzWqXPaO2pW8YDAhpQk
	 AhPhmPjxSblrV9w1iZDnohvWyorZS4i41I1DhHJ+B7p4E9TN5TbiSPgi/aLU6XB9uXkhfC
	 8mTxEdH4xO54gVEqURLKEgXtiUc8LNne6uYBdrXyQVZE0YpabWecJShr8xGUi2F4Cgm8J8
	 8D1IqQ4hvMx7uiEFE4pihggXIYvDHhk3bUzk5vqMHffUPy/peGatAwflQmeZK/0jnH0CGo
	 2i20Ya3ICpMQA7W7hVYeAEHAGArNsINL81TsFKE4RoA==
ARC-Authentication-Results: i=1; mx.naver.com; 
  spf=pass (mx.naver.com: domain of kossw123@gmail.com designates 209.85.214.175 as permitted sender) smtp.mailfrom=kossw123@gmail.com;
  dkim=pass header.i=@gmail.com;
  dmarc=pass (p=NONE sp=QUARANTINE dis=NONE) header.from=gmail.com
Return-Path: <kossw123@gmail.com>
Received-SPF: pass (mx.naver.com: domain of kossw123@gmail.com designates 209.85.214.175 as permitted sender)
  client-ip=209.85.214.175; x-iptype=white;
Authentication-Results: mx.naver.com;
  spf=pass (mx.naver.com: domain of kossw123@gmail.com designates 209.85.214.175 as permitted sender) smtp.mailfrom=kossw123@gmail.com;
  dkim=pass header.i=@gmail.com;
  dmarc=pass (p=NONE sp=QUARANTINE dis=NONE) header.from=gmail.com
X-Naver-ESV: +deYpB3G1H+qbrJmjAKwjAuwKxEmaBwBjg==
X-Session-IP: 209.85.214.175
Received: from mail-pl1-f175.google.com (mail-pl1-f175.google.com [209.85.214.175])
  by crcvmail306.nm.naver.com with ESMTP id O15yZ67UR9O06ILR-1ag9g
  for <kossw123@naver.com>
  (version=TLSv1.3 cipher=TLS_AES_128_GCM_SHA256);
  Wed, 18 Jun 2025 11:15:09 -0000
Received: by mail-pl1-f175.google.com with SMTP id d9443c01a7336-23636167b30so60837945ad.1
        for <kossw123@naver.com>; Wed, 18 Jun 2025 04:15:08 -0700 (PDT)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
        d=gmail.com; s=20230601; t=1750245308; x=1750850108; darn=naver.com;
        h=content-transfer-encoding:mime-version:references:in-reply-to
         :message-id:date:subject:to:from:from:to:cc:subject:date:message-id
         :reply-to;
        bh=UU2pyKWcL22myledRTIjWFL5EhBsM4P4oVJOu/Hxj2A=;
        b=IPFE/H2VAzQzyHvRrtM2cN+ap3Cqas16LlzrOfBAykwaf4N/PuNaSqcT5JHwUCycLe
         mUOOxyL9obWSMSkJZZyM+/pczd6fum2oynGTqL4vyNKoXdef/xObo7XJv25DJJmsQ+qo
         +J+8ci52NBmJ0XZYb0ZSLD1qKhH0CnVolGsXTvyWvis2M/T55pA91v1TLFVmfZ3u27IX
         UZCkKRG8zEuOEB8b/BXLIxBlkzWkrR7CptGH5BeobAN1rD/m+myX3nlVlgCRdh80wPzW
         /Ni1qEhuIBN9y1tXHl6SLBTTc0PBlfO59gWeOvvoiSWgELc/8fz5FEOt2ARwWyp+pKxh
         MRKg==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
        d=1e100.net; s=20230601; t=1750245308; x=1750850108;
        h=content-transfer-encoding:mime-version:references:in-reply-to
         :message-id:date:subject:to:from:x-gm-message-state:from:to:cc
         :subject:date:message-id:reply-to;
        bh=UU2pyKWcL22myledRTIjWFL5EhBsM4P4oVJOu/Hxj2A=;
        b=Mc7Cc5KAR0y9gVYeObVH8q3F2vc0zlFK6QtqIClu7/TQviuMrwQrmusn6YokSn5bSV
         y+TR+CU7mr9J5q3XkkgLFtmylaJHNXwGSXH2WfkHbEYUZ0Fl+8m2hvKkn9zVHJkQ8g1p
         KAompdbyC8813zuRocYHNiJ3h1Fayek9q/MzHq6/eg8w8gjIBnZK+qTEWuBCwors3mpS
         PSYszIhjdDuebkPRa7zZdea7H2od7VA9GVrCrSmPzzHq8okU75Ndiu6ydp06m+AM0iqR
         i7O1I1vuNJhRmBNMfiXRe021TXu3VxB1WPh4X3AEyts8XssTkG32FdjxYlvyGJj5z1Xh
         yr/Q==
X-Gm-Message-State: AOJu0YyxPOgh22DEp661iINR5JK5x+vfOfvxo3fTqcKUDGPP7iegMeSY
	JV3fS+EYJKzZW4OFZJo419PIoByjO/CxbBalZMUliEk6lK1yskeL0Ny9c1efjg==
X-Gm-Gg: ASbGncuawIwB6TCqGyLkH/Mxh6CrWNvrYjWHaA5boULnSyTFnaaci2SYNzF4Ob1HRfC
	+YbXa1QLgZMYk2KgBCXhIk690S6ViFFknHZfWP132vyEiBBU9rF5fPjYPKEgW0YUVIZj2lM1Wno
	QY85ronPz4qBwnvn5upw827S0znHsXWMmjOpjZNTRM2VpFFE6w2lFK3R4o2hDCRhdhx/eIDuOx1
	mnXs3Ttwc0L69cZFJHVMqC3AajGn8Hyt3yGt6nAVocDZrv8nubJOFbWbK3PuL6f9DwVgzOsN239
	6n4D6b0sq67cwbwe0urBkkmVemRxwUJyfIRA6DbnzZO80AITlsv+DOv9S2bbAmw3SYlvxjocUbw
	NdO6PV5HRfPPnVW0=
X-Google-Smtp-Source: AGHT+IHI8hPL7NH5gs62HJCtr0NMkZEGEBG4WpO9LtXR/SxacnKXEtJA6HScBxiDLRnYLYzq32YqgQ==
X-Received: by 2002:a17:902:ecce:b0:22f:b6d6:2737 with SMTP id d9443c01a7336-2366b32e726mr288026465ad.10.1750245307789;
        Wed, 18 Jun 2025 04:15:07 -0700 (PDT)
Return-Path: <kossw123@gmail.com>
Received: from localhost.localdomain ([1.229.44.19])
        by smtp.gmail.com with ESMTPSA id d9443c01a7336-2365decb5f9sm97068745ad.194.2025.06.18.04.15.06
        for <kossw123@naver.com>
        (version=TLS1_3 cipher=TLS_AES_256_GCM_SHA384 bits=256/256);
        Wed, 18 Jun 2025 04:15:06 -0700 (PDT)
From: Your Name <kossw123@gmail.com>
To: kossw123@naver.com
Subject: [PATCH] testing
Date: Wed, 18 Jun 2025 20:14:45 +0900
Message-Id: <20250618111446.1444-1-kossw123@gmail.com>
X-Mailer: git-send-email 2.39.0.windows.2
In-Reply-To: <y>
References: <y>
MIME-Version: 1.0
Content-Type: text/plain; charset=y
Content-Transfer-Encoding: 8bit

From: 8iiow <kossw123@naver.com>

---
 test.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/test.txt b/test.txt
index 0ef79b8..e7a4544 100644
--- a/test.txt
+++ b/test.txt
@@ -7,4 +7,5 @@ anskdonasdkonsnadkosandklosandklsandlksa
 
 asdnioasndosandosano
 
-testing
\ No newline at end of file
+testing
+add 하고나서 해보기
\ No newline at end of file
-- 
2.39.0.windows.2

